packages/zod/src/v4/core/checks.ts

```diff
@@ -779,8 +780,15 @@ export const $ZodCheckStringFormat: core.$constructor<$ZodCheckStringFormat> = /
     $ZodCheck.init(inst, def);
 
     inst._zod.onattach.push((inst) => {
-      inst._zod.bag.format = def.format;
-      if (def.pattern) inst._zod.bag.pattern = def.pattern;
+      const bag = inst._zod.bag as $ZodStringInternals<unknown>["bag"];
```

Comment:
I struggled to find where `bag` was getting changed due to these referencing the `bag: Record<string, unknown>;` type